docs/source/snapshot.rst

```diff
@@ -38,24 +78,83 @@ from ``long_name`` to its values.
     possible to choose values which shouldn't be available on the respective
     ValueAttribute.
 
+.. _requirement_types:
+
 Requirement Types (``requirement_types``)
 =========================================
 
+.. code-block:: yaml
+
+   requirement_types: # WorkItemTypes
+    system_requirement:
+      long_name: System Requirement
+      attributes: # Field Definitions, we don't need the IDs
+        Capella ID: # Field name
+          type: String # -> AttributeDefinition
+        Type:
+          type: Enum
+        Submitted at:
+          type: Date # -> AttributeDefinition
+        Release:
+          type: Enum
+          multi_values: true
+
+    software_requirement:
+      long_name: Software Requirement
+      attributes:
+        Capella ID:
+          type: String
+        Type:
+          type: Enum
+        Submitted at:
+          type: Date
+
+    stakeholder_requirement:
+      long_name: Stakeholder Requirement
+      attributes:
+        Capella ID:
+          type: String
+
 Polarion supports work item types as a special field. This section is therefore
 a mapping that describes ``RequirementType``\ s from a given ``identifier`` to
 its ``long_name`` and ``attribute_definitions`` (in short ``attributes``).
-Therein the keys are matched against the ``data_types`` if it is an
-``AttributeDefinitionEnumeration``.
+Therein the keys are matched against the ``long_name`` of the
+``EnumDataTypeDefinition`` defined in ``data_types`` if it is an
+``AttributeDefinitionEnumeration``. Else an ``AttributeDefinition`` is meant
+and for these a type-hint via ``type`` is needed.
```

Comment:
No it is mandatory. We don't check if the key is in the `data_types` section to check that it's an `EnumerationAttributeDefinition`. We go by the `type: Enum` hint. We could make the Interface more slim by implementing this logic. But we need to anticipate the ID change first: Instead of using names as keys for matching, IDs should be taken.  What do you think?